### PR TITLE
replaced user-guide link with quick-start-guide

### PR DESCRIPTION
**Pull request by peteroneilljr**
Just fixing a broken link on the index page. I also cleaned up some of the markdown. If there's a specific markdown style guide you're following, let me know and I can adjust it.

Cheers,
Peter

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
ideally, the change should have only touched the links and omit formatting changes in the diff.

File: site/content/docs/_index.md [hunk 1]
```diff
@@ -4,25 +4,22 @@ TODO(bentheelder): this page should probably be removed now that we have navigat
...
-For those interested in contributing to the project with code or documentation,
-the [development guide][dev guide] will give you a more in-depth look into how 
-kind works. 
-We will start by getting the required software and setting up the environment
-and then take a look into how kind and its different packages are laid out.
+If you are interested in modifying or contributing to kind, you can go and checkout our [design docs][design] where we explain how things are implemented.
+
+For those interested in contributing to the project with code or documentation, the [development guide][dev guide] will give you a more in-depth look into how kind works.
+
+We will start by getting the required software and setting up the environment and then take a look into how kind and its different packages are laid out.
```
Comment (neolit123) [inline site/content/docs/_index.md:18]: the length of the above lines should have been kept.

**Comment by peteroneilljr** [comment]
No problem, I added the line breaks back.

**Comment by BenTheElder** [comment]
just curious: did you find an inbound link to this page somewhere? I'd forgotten it exists, I don't _think_ we're actually using it.

(we should still fix it!)

**Pull request merged**